models/hospital.go

```diff
@@ -0,0 +1,7 @@
+package models
+
+type Hospital struct {
+	ID               int    `json:"id" db:"id"`
+	Name             string `json:"name" db:"name"`
+	MaxPatientAmount int    `json:"max_patient_amount" db:"max_patient_amount"`
```

Comment:
The json representation should be `camelCase`. Do this for every models' attributes.

Reply:
Using `camelCase` doesn't export those fields to outside packages, Also trying to do that is returning the error message: `scannable dest type struct with >1 columns (n) in result` and reported in code inspection by GoLand too. Going with capitalized first letter :)

Reply:
@PrathameshMutkure I said the json representation. 
```suggestion
	MaxPatientAmount int    `json:"maxPatientAmount" db:"max_patient_amount"`
```
for example.

Reply:
Oh yeah got it :)
I'll do it